### ISSUE TITLE
HOWTO host phoenix_guides locally and browse when on the road

### ISSUE THREAD

**Issue by 0xMF** [phoenixframework/phoenix_guides#283]
I'm new to phoenix. I want to learn phoenix using [phoenix_guides](https://github.com/phoenixframework/phoenix_guides). How do I host these guides locally and browse them as I will be on the road much. Any help would be much appreciated.

Of course, I am comfortable with reading the actual markdown files directly but I'd rather get the rendered html with linking using the style sheets when I'm learning and on the road.

Thanks! :smile:

**Comment by 2kodes** [comment]
hi @0xMF ,

 if you have wget installed you can use the following to get or download the www.phoenixframework.org  site so you can have it online. ;)

wget  --mirror  --page-requisites  --convert-links  --recursive  --adjust-extension -p -w 2  -e robots=off  --no-check-certificate  --html-extension   -t 100000   --no-cookies  www.phoenixframework.org/
Since most of the time the website is updated with info from the phoenix_guides repo, you could just rerun the above in case of any updates..

**Comment by lancehalvorsen** [comment]
@2kodes, you were replying just as I was. :)

@0xMF, if you're just interested in the content and links aren't that important, you can clone the repo and use a markdown editor to read the files.

**Comment by 0xMF** [comment]
@2kodes @lancehalvorsen thank you for your responses.  

Yes, I have read these markdown files using vim and I know I can download the phoenix docs website itself, but I wanted to know whether phoenix itself (or one of the tools bundled with phoenix) could be used to render the markdown files in phoenix_guides. So what I am asking is

1. install phoenix (done)
2. clone phoenix_guides (done)
3.  ?? (what phoenix/cowboy/elixir command could I use to render phoenix_guides from the local cloned repo so I'm running these docs locally but it looks and feels as if I were connected to the [actual phoenix website documentation](http://www.phoenixframework.org/v0.13.1/docs) because the content over there is coming from this repo, right?)

Hopefully, I clarified what I was asking...

**Comment by lancehalvorsen** [comment]
@0xMF ah, I see. We use http://readme.io/ for the phoenixframework.org site. readme.io allows us to input markdown files along with some custom css into their application, which transforms it into the site you see.

We don't have a Phoenix/Cowboy/Elixir command to do this at the moment, but I would love it if one existed. :^)

**Comment by 0xMF** [comment]
@lancehalvorsen Heh! I thought phoenix was used everywhere...lol so I expected there would be a phoenix command to render these docs that I didn't know about. 

Anyway, since these are markdown files I found example 3 in [pandoc](http://pandoc.org/demos.html) to be good enough to generate html files. In fact I could create a decent html page using the following command

```shell
pandoc -s -S --toc -c pandoc.css  B_adding_pages.md -o B_adding_pages.html
```
While not an exact duplicate, the styling and so on is good enough to read. I'll play around a bit more with pandoc to get something I can use but for now I think I got my answer. Thanks again for all your help.
